src/patternfly/components/Page/page.scss

```diff
@@ -453,6 +466,35 @@ $pf-c-page--height-breakpoint-map: build-breakpoint-map("base", "sm", "md", "lg"
       width: 100%;
     }
   }
+
+  &.pf-m-nav {
+    --pf-c-page__sidebar-body--PaddingTop: var(--pf-c-page__sidebar-body--m-nav--PaddingTop);
+    --pf-c-page__sidebar-body--PaddingBottom: var(--pf-c-page__sidebar-body--m-nav--PaddingBottom);
+  }
+
+  &.pf-m-inset {
+    padding-right: var(--pf-c-page__sidebar-body--m-inset--PaddingRight);
+    padding-left: var(--pf-c-page__sidebar-body--m-inset--PaddingLeft);
+  }
+
+  &.pf-m-no-inset {
+    padding-right: 0;
+    padding-left: 0;
+  }
+
+  &.pf-m-no-padding {
+    padding-top: 0;
+    padding-bottom: 0;
+  }
+
+  &.pf-m-padding {
```

Comment:
if this is the default, what's the use case for this class? I would probably think a sidebar body element has no padding by default from the way you've set everything up here.

Reply:
Just the inverse if padding needs to be reapplied